CMakeLists.txt

```diff
@@ -254,87 +253,109 @@ else()
 	set(_library_dir lib)
 endif()
 
-set(INCLUDE_INSTALL_ROOT_DIR include)
+set(INCLUDE_INSTALL_ROOT_DIR ${CMAKE_INSTALL_PREFIX}/include)
```

Comment:
Don't do this, see http://cgold.readthedocs.io/en/latest/tutorials/install/install-prefix.html